website/src/pages/showcase/index.tsx

```diff
@@ -328,11 +325,19 @@ function ShowcaseCards() {
 }
 
 export default function Showcase(): JSX.Element {
+  const filteredUsers = useFilteredUsers();
   return (
     <Layout title={TITLE} description={DESCRIPTION}>
       <main className="margin-vert--lg">
         <ShowcaseHeader />
         <ShowcaseFilters />
+        {filteredUsers.length !== 0 && (
+          <div
+            style={{display: 'flex', marginLeft: 'auto'}}
+            className="container">
+            <SearchBar />
+          </div>
+        )}
```

Comment:
I don't think this is good. The `<SearchBar>` should remain the same either if there are results or not.